### DISCUSSION TITLE
How to shift equations position to fit long equations with MathJax?

### DISCUSSION THREAD

**Discussion by victorfeagins** [quarto-dev/quarto-cli#6279] [Q&A] [answered]
### Description

I saw in this Q&A how to change math font size increase : https://github.com/quarto-dev/quarto-cli/discussions/2392
But how do I shift the equation to make it left aligned to not go off the screen? I tried using the .MathJax_Display but it did not work
<img width="1196" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/67335904/50f7a68d-fd59-4212-81ed-696e06c2010b">

```
---
title: "equations"
format: html

html-math-method: mathjax
---
```{=html}
<style>
.MathJax {
    font-size: 200% !important;
}
.MathJax_Display { 
    text-align: left !important;
}

</style>
```

```
$$
\tau  (x1,x2,c1) = .228 + .05 \cdot \mathbb{1}(x_1 < .07) - .05 \cdot\mathbb{1}(x_2 < -.69) - .08 \cdot \mathbb{1}(c_1 \in \{1, 13, 14\})
$$
```

Text

**Comment by mcanouil** [comment, marked as answer]
See <https://quarto.org/docs/authoring/article-layout.html> and/or use your favourite browser inspect mode to figure out what selector to use to customise via CSS.

**Comment by victorfeagins** [comment]
Yep I was able to wrap my equations. Thank you!

:::{.column-page}
$$
my equation
$$
:::
